File: app/src/main/java/com/project/scratchstudio/kith_andoid/GetInternalData/GetCountData.java
package com.project.scratchstudio.kith_andoid.GetInternalData;

import android.content.SharedPreferences;
import android.util.Log;

import com.project.scratchstudio.kith_andoid.Model.Cache;

public class GetCountData implements IGetInternalData {

    @Override
    public int get( SharedPreferences sp) {
        Cache.setAllUsers(sp.getInt("count_users", 0));
        Log.i("INTERNAL GET: ", "I'm here: " + Cache.getAllUsers());
        return 0;
    }

}

File: app/src/main/java/com/project/scratchstudio/kith_andoid/GetInternalData/GetCountData.kt
package com.project.scratchstudio.kith_andoid.GetInternalData

import android.content.SharedPreferences
import android.util.Log

import com.project.scratchstudio.kith_andoid.model.Cache

class GetCountData : IGetInternalData {

    override fun get(sp: SharedPreferences) {
        Cache.allUsers = sp.getInt("count_users", 0)
        Log.i("INTERNAL GET: ", "I'm here: " + Cache.allUsers)
    }

}
